modules/tensor_mechanics/test/tests/cp_user_object/linesearch.i

```diff
@@ -164,49 +142,43 @@
 [Materials]
   [./crysp]
     type = FiniteStrainUObasedCP
-    block = 0
     stol = 1e-2
     tan_mod_type = exact
-    maximum_substep_iteration = 10
+    maximum_substep_iteration = 200
+    use_line_search = true
+    min_line_search_step_size = 0.01
```

Comment:
What and why?

Reply:
wait, what?! I did not make those changes. I wonder if git renames are confusing the differ because all files in the PR are so similar